oppia_android_test.bzl

```diff
@@ -0,0 +1,45 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+def oppia_android_test(name, srcs, test_manifest, custom_package, resource_files,
+                       test_class, src_library_name):
+  '''
+  This macro exists as a way to set up a test in Oppia Android to be run with Bazel.
+  This macro creates a library for an individual test that is fed as a dependency into an
+  android_local_test() rule which configures the test to be run with Roboletric and Bazel.
+
+  Args:
+      name: str. The name of the Kotlin test file without the '.kt' suffix.
+      srcs: str. The name of the Kotlin test file to be run.
```

Comment:
Is this correct? Isn't this supposed to be a list of Kotlin source files that include the test file being run?

Reply:
Fixed!